be/src/olap/iterators.h

```diff
@@ -28,27 +28,63 @@ class RowBlockV2;
 class Schema;
 class Conditions;
 
-struct StorageReadOptions {
-    // lower_bound defines the smallest key at which iterator will
-    // return data.
-    // If lower_bound is null, won't return
-    std::shared_ptr<RowCursor> lower_bound;
-
-    // If include_lower_bound is true, data equal with lower_bound will
-    // be read
-    bool include_lower_bound = false;
-
-    // upper_bound defines the extend upto which the iterator can return
-    // data.
-    std::shared_ptr<RowCursor> upper_bound;
-
-    // If include_upper_bound is true, data equal with upper_bound will
-    // be read
-    bool include_upper_bound = false;
-
-    // reader's column predicates
-    // used by zone map/bloom filter/secondary index to prune data
-    std::shared_ptr<Conditions> conditions;
+class StorageReadOptions {
```

Comment:
I prefer keep all `Options` as struct. Because most `Options` operations are getter and setter. if we make these fields as private, it will produce a lot of getter and setter which I think is not necessary.
Let's keep it as a `struct`.

Reply:
ok